tests/ui/unconventional_macro_braces.stderr

```diff
@@ -0,0 +1,29 @@
+error: use of irregular braces for `vec!` macro
+  --> $DIR/unconventional_macro_braces.rs:5:13
+   |
+LL |     let x = vec! {1, 2, 3};
+   |             ^^^^^^^^^^^^^^
+   |
+   = note: `-D clippy::unconventional-macro-braces` implied by `-D warnings`
+help: consider writing `vec! [1, 2, 3]`
```

Comment:
The space between `!` and the opening brace is only standard style for curly braces, would be nice if the suggestion could take that into account.